uicomponent-compose/feed/src/main/java/io/github/droidkaigi/feeder/feed/FeedViewModel.kt

```diff
@@ -38,10 +38,8 @@ private val LocalFeedViewModel = compositionLocalOf<FeedViewModel> {
     error("not LocalFeedViewModel provided")
 }
 
-@Composable
-fun ProvideFeedViewModel(viewModel: FeedViewModel, block: @Composable () -> Unit) {
-    CompositionLocalProvider(LocalFeedViewModel provides viewModel, content = block)
-}
+fun feedViewModelProviderValue(viewModel: FeedViewModel) =
+    LocalFeedViewModel provides viewModel
```

Comment:
Replaced ProvideViewModel Composable with a function that creates a ProvidedValue.